demo/OrderCloud.DemoWebApi/AppSettings.cs

```diff
@@ -8,5 +8,7 @@ namespace OrderCloud.DemoWebApi
 	public class AppSettings
 	{
 		public string MyDBConnectionString { get; set; }
-	}
+
+		public string WebhookHashKey { get; } = "myhashkey"; 
```

Comment:
You could have a comment on this property like "Should match the HashKey configured on your webhook in the Ordercloud portal" or something.